app/config.c

```diff
@@ -87,6 +86,9 @@ error_mode_t    error_mode = ERROR_MODE_NONE;
 
 cpu_state_t     cpu_state[MAX_CPUS];
 
+core_type_t     hybrid_core_type[MAX_CPUS];
```

Comment:
This - and cpu_state - ought to be arrays of uint8_t, so as not to waste 3 * MAX_CPU bytes. Last time I checked the C standard, the C++11+ typed enums hadn't been imported to C, though :'(

Reply:
Last time I checked, GCC optimised the storage size to be a single byte per element.

Reply:
So it does, indeed. TIL.
I'm not fully happy with reliance on such kinds of special optimizing compiler behaviour, but as long as it works, after all...